include/openssl/opensslconf.h.in

```diff
@@ -0,0 +1,15 @@
+#if defined(_MSC_VER) && !defined(__clang__) && !defined(__attribute__)
+#define __attribute__(a)
+#endif
+
+#if defined(LIBRESSL_INTERNAL) && !defined(LIBRESSL_NAMESPACE)
+# define LCRYPTO_UNUSED(x)
+# define LCRYPTO_USED(x)
+# define LCRYPTO_ALIAS1(pre, x)
+# define LCRYPTO_ALIAS(x)
+
+# define LSSL_UNUSED(x)
+# define LSSL_USED(x)
+# define LSSL_ALIAS(x)
```

Comment:
I suppose this is acceptable as a workaround for now... What makes this uglier than I'd like it to be is that these are now exposed in a public header (yes, they're guarded by LIBRESSL_INTERNAL, but still) while they should really only be internal.

The defines in this block should eventually be picked up from `libcrypto/hidden/crypto_namespace.h` and `libssl/hidden/ssl_namespace.h`.

Reply:
Thanks! I removed this and instead switched to pulling in the 'hidden' headers for libcrypto and libssl.